examples/api-deploy-gke-nebius-okta/README.md

```diff
@@ -0,0 +1,296 @@
+| `--set gcpCredentials.enabled=true` | Enables GCP credentials |
+| `--set gcpCredentials.projectId=$GCP_PROJECT_ID` | GCP project ID |
+| `--set gcpCredentials.serviceAccountJson=$GCP_SERVICE_ACCOUNT_JSON` | GCP service account JSON credentials |
+| `--set nebiusCredentials.enabled=true` | Enables Nebius credentials |
+| `--set nebiusCredentials.tenantId=$NEBIUS_TENANT_ID` | Nebius tenant ID |
+</details>
+
+
+
+## Step 3: Get the endpoint and configure your DNS
+
+```bash
+HOST=$(kubectl get svc ${RELEASE_NAME}-ingress-nginx-controller --namespace $NAMESPACE --kubeconfig $TMP_KUBECONFIG --context $GKE_CONTEXT -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+ENDPOINT=http://$HOST
+echo $ENDPOINT
+```
+
+If you see a blank address, wait a bit and try again. The GCP load balancer takes 1-2min to get an external IP.
+
+Configure your DNS to point to the IP address of the API server. This is required for Okta to verify the `redirect_uri` configured by you in the Okta app. Alternatively, update the `redirect_uri` in the Okta app to use the IP address of the API server.
+
+Try opening the endpoint in a browser. You should see the SkyPilot dashboard login page. 
+
+<div align="center">
+  <img src="https://i.imgur.com/k17TpJU.png" width="500px" alt="Okta login page">
+</div>
+
+If the login page shows 503 error, make sure the API server pod is healthy:
+```bash
+kubectl get pods --namespace $NAMESPACE --kubeconfig $TMP_KUBECONFIG --context $GKE_CONTEXT
+```
+
+## Step 4: Configure the CLI and launch your first job
+
+On your client(s), install the SkyPilot CLI:
+```bash
+pip install -U skypilot-nightly
+```
+
+Login to the API server:
+```bash
+sky api login -e $ENDPOINT # E.g., http://34.42.25.204 or http://sky.yourorg.com
+```
+
+A browser will open and you will be redirected to the Okta login page. Login with your Okta credentials. You will receive a token:
+
+<div align="center">
+  <img src="https://i.imgur.com/OflAVTp.png" width="500px" alt="Okta token page">
+</div>
+Copy the token and paste it in the CLI. You should see the following message:
+
+```console
+$ sky api login -e http://sky.yourorg.com
+Authentication is needed. Please visit this URL setup up the token:
+
+http://sky.yourorg.com/token
+
+Opening browser...
+Paste the token: 
+```
+
+Run `sky check` to verify cloud setup:
+```console
+$ sky check
+...
+🎉 Enabled infra 🎉
+  GCP [compute, storage]
+  Kubernetes [compute]
+    Allowed contexts:
+    ├── nebius-cluster
+    └── gke-cluster
+  Nebius [compute]
+```
+
+## 🎉 SkyPilot API server is ready to use!
+
+Some commands to try:
+
+* `sky dashboard` to open the dashboard in your browser
+* `sky launch -c test --gpus H100:1 -- nvidia-smi` to launch a job with 1 H100 GPU
+* `sky show-gpus` to show available GPUs
+* `sky status` to see SkyPilot status and infra available
+
+## ✨ Bonus: Use Nebius shared filesystem with SkyPilot
```

Comment:
Can we add the infiniband example as well?

Reply:
good call, added!